src/components/Lobby/Lobby.jsx

```diff
@@ -27,24 +28,28 @@ export class Lobby extends React.Component {
                     </div>
                 </div>
                 <div className={styles.rooms}>
-                    <Button cy="create-room" handleClick={this.props.createRoom} text="Create Room" />
+                    <div className={styles.createRoom}>
+                        {!this.props.number ? <h1>Create room: </h1> : <Button cy="create-room" handleClick={() => this.props.createRoom(this.props.number)} text="Create Room" />}
+                        {!this.props.number && this.props.createRoom && <RoundCount />}
```

Comment:
`&& this.props.createRoom` isn't doing anything here as it should always be getting that function from redux I think?